index.js

```diff
@@ -18,4 +20,339 @@ const openMenu = document.querySelector('.button');
+      arr[0].image}<div class = "popbox2"><p class = "popuppara">${arr[0].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+function click1() {
+  if (button1.id == 'probutton') {
+    div.innerHTML = `<h2 class="popupname">${arr[1].name.innerHTML}</h2><ul class="popuplist">${arr[1].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[1].image}<div class = "popbox2"><p class = "popuppara">${arr[1].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+function click2() {
+  if (button2.id == 'probutton2') {
+    div.innerHTML = `<h2 class="popupname">${arr[2].name.innerHTML}</h2><ul class="popuplist">${arr[2].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[2].image}<div class = "popbox2"><p class = "popuppara">${arr[2].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+function click3() {
+  if (button3.id == 'probutton3') {
+    div.innerHTML = `<h2 class="popupname">${arr[3].name.innerHTML}</h2><ul class="popuplist">${arr[3].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[3].image}<div class = "popbox2"><p class = "popuppara">${arr[3].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+function click4() {
+  if (button4.id == 'probutton4') {
+    div.innerHTML = `<h2 class="popupname">${arr[4].name.innerHTML}</h2><ul class="popuplist">${arr[4].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[4].image}<div class = "popbox2"><p class = "popuppara">${arr[4].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+function click5() {
+  if (button5.id == 'probutton5') {
+    div.innerHTML = `<h2 class="popupname">${arr[5].name.innerHTML}</h2><ul class="popuplist">${arr[5].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[5].image}<div class = "popbox2"><p class = "popuppara">${arr[5].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+function click6() {
+  if (button6.id == 'probutton6') {
+    div.innerHTML = `<h2 class="popupname">${arr[6].name.innerHTML}</h2><ul class="popuplist">${arr[6].technologies.innerHTML}</ul><div class = "popbox">${
+      arr[6].image}<div class = "popbox2"><p class = "popuppara">${arr[6].description.innerHTML}</p><div class = "popbox3">${link.innerHTML}${source.innerHTML}</div></div></div>`;
+  }
+}
+
+button0.addEventListener('click', click0);
+button1.addEventListener('click', click1);
+button2.addEventListener('click', click2);
+button3.addEventListener('click', click3);
+button4.addEventListener('click', click4);
+button5.addEventListener('click', click5);
+button6.addEventListener('click', click6);
```

Comment:
- **[OPTIONAL]** Kindly refactor your code to avoid repetitive code, remember DRY (don't repeat yourself) for the better practices of JavaScript. 👍🏽